packages/rum-react/src/domain/reactRouterV6/startReactRouterView.ts

```diff
@@ -33,13 +35,27 @@ export function computeViewName(routeMatches: RouteMatch[]) {
       if (!viewName.endsWith('/')) {
         viewName += '/'
       }
-      if (path === '*') {
-        viewName += routeMatch.params['*']!
-      } else {
-        viewName += path
-      }
+      viewName += path
     }
   }
 
   return viewName
 }
+
+function substitutePathWildcards(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I don't fully understand what this function does. Can you add a comment maybe with some path example before and after?